app/src/util/constants.ts

```diff
@@ -105,8 +105,10 @@ export const PermissionUriMap: { [key: string]: string[] } = {
     '/lnrpc.Lightning/SendCoins',
     '/lnrpc.Lightning/SendMany',
     '/lnrpc.Lightning/SendPayment',
+    '/lnrpc.Lightning/SendPaymentV2',
     '/lnrpc.Lightning/SendPaymentSync',
     '/lnrpc.Lightning/SendToRoute',
+    '/lnrpc.Lightning/SendToRouteV2',
```

Comment:
These methods are in the `routerrpc` package, so the uris need to be `/routerrpc.Router/SendPaymentV2` and `/routerrpc.Router/SendToRouteV2`.

Reply:
Ouch... How did I miss that? Shouldn't do drive-by reviews early in the morning :see_no_evil:

Reply:
😂

I would have missed it too.

Reply:
Thanks for testing! I wonder why `SendPayment` is working then because it's also in the `routerrpc` package but is set as `lnrpc` here... 🤔 Or is that also not working?

Reply:
No, there are actually 3 `SendPayment` RPC methods... The oldest in `lnrpc` which still can be used but lacks features, the intended replacement in `routerrpc` that was deprecated even while the `routerrpc` package was still labelled as experimental, and then the one that everybody should use: `SendPaymentV2` in `routerrpc`.
Unfortunately that's what you get if you promise your users to never break RPC methods, you need to write new ones if you want to change things...

Reply:
Ah ok got it, do you want me to also add the ` /routerrpc.Router/SendPayment` one then?

Reply:
No, since that should definitely not be used by anyone.

Reply:
It is updated now - sorry I should have noticed that this was under the wrong path I think but I just assumed it was the same as the original one.